projects/components/src/form-field/form-field.component.ts

```diff
@@ -25,6 +26,7 @@ import { IconSize } from '../icon/icon-size';
           'show-border': this.showBorder,
           'error-border': this.showFormError && this.errorLabel
         }"
+        [ngStyle]="{ 'background-color': this.contentBgColor }"
```

Comment:
Couldn't we work with `background-color: inherit` so that we don't have to rely on inputs to enable us to style items?

Reply:
content block is a child inside the form-field . so, inherit doesn't work.

Reply:
alternative is ng-deep. I prefer this over that.